### ISSUE TITLE
[BUG] kvrocks2redis was unable to sync to other Kvrocks's namespaces except default

### ISSUE THREAD

**Issue by xyxu** [apache/kvrocks#489]
redis_writer.cc:115] [kvrocks2redis] CRITICAL - redis sync return error , administrator confirm needed : -administor permission required to perform the command

**Comment by git-hulk** [comment]
Yes, thanks for your feedback. We should use `flushdb` instead of `flushall` when flushing the target db.

**Comment by caipengbo** [comment]
I'll fix it as soon as possible.

**Comment by caipengbo** [comment]
What @git-hulk  mentioned is indeed a problem, but I don't think it is the cause of your problem. Can you provide more information (Your `kvrocks2redis` configuration; Your behavior at the `Kvrocks`) @xyxu
